src/system_metrics/mongoose_system_metrics_collector.erl

```diff
@@ -119,11 +133,14 @@ get_components() ->
     Domains = mongoose_router:get_all_domains() ++ ejabberd_router:dirty_get_all_components(all),
     Components = [ejabberd_router:lookup_component(D, node()) || D <- Domains],
     LenComponents = length(lists:flatten(Components)),
-    #{report_name => cluster, key => number_of_components, value => LenComponents}.
+    #{components => LenComponents}.
 
 get_api() ->
     ApiList = filter_unknown_api(get_http_handler_modules()),
-    [#{report_name => http_api, key => Api, value => enabled} || Api <- ApiList].
+    [#{name => http_api, params => 
+        lists:foldl(fun(Element, Acc) ->
```

Comment:
Minor: Unexpected indentation, I would either break before `, params` or move it right after `, params =>`